### DISCUSSION TITLE
DLT Message Analyzer installation help

### DISCUSSION THREAD

**Discussion by firedevil79** [COVESA/dlt-viewer#317] [Q&A] [answered]
Hi Team, 

I am using DLT viewer quite a lot and I am very much interested to get also the additional Plugins running.
I was able to compile the DLT viewer via console for Windows.
but, when I am trying to follow the guide for compiling via QT IDE, I get an error finding third party libraries..C:\Qt\Tools\CMake_64\share\cmake-3.24\Modules\FindPackageHandleStandardArgs.cmake:230: Fehler: Could NOT find ANTLR (missing: Java_JAVA_EXECUTABLE) C:/Qt/Tools/CMake_64/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE) plugin/DLT-Message-Analyzer/dltmessageanalyzerplugin/src/cmake/FindANTLR.cmake:121 (find_package_handle_standard_args) plugin/DLT-Message-Analyzer/dltmessageanalyzerplugin/src/CMakeLists.txt:63 (find_package)

I guess, I am doing something wrong with the folders?
Anyone made the compiling recently and got it working?

Any help or guidance would be awesome.. thanks in advance 
Jens

**Comment by firedevil79** [comment]
![Screenshot 2022-12-20 200741](https://user-images.githubusercontent.com/70649483/208747266-4f0e63ec-a812-46d0-9a9e-cff1e8a707d3.png)
![Screenshot 2022-12-20 200639](https://user-images.githubusercontent.com/70649483/208747269-e94a2f85-bcab-4965-9b0e-f6d22e584543.png)

**Comment by firedevil79** [comment, marked as answer]
hmm. looks like Java installation is also needed as requirement. 
I Installed Java SDK and at least now it continues 
Ill drop message when I know more for fellow people ;)

**Reply by firedevil79** [reply]
this was the point, Java SDK is required, then it builds successful

**Comment by leo-lu-niz** [comment]
Hi Team,

I am using DLT viewer 2.20 vision, and I set filter to only receive  DEBUG message, but I have no idea why I could not receive DEBUG message?

I could receive other levels， so I guess it is not the connection problem.

Any help or guidance would be awesome.. thanks in advance.

Leo
